### DISCUSSION TITLE
Make `isNumber` return true for NaN

### DISCUSSION THREAD

**Discussion by aleclarson** [radashi-org/radashi#74] [Ideas]
When a NaN value is regrettably propagated through a program, it's best to run it through code that expects to be working with numbers (which happens to be code guarded by `isNumber`). That way, the NaN value has a better chance of not crashing the program, and it's more likely to be coerced into a non-NaN value.

### Related issues

https://github.com/sodiray/radash/issues/405

**Comment by MarlonPassos-git** [comment]
https://github.com/radashi-org/radashi/pull/248
